app/metricslogger.py

```diff
@@ -206,8 +223,27 @@ def capture_vm_metrics(self):
                 vmmetrics = self.get_metrics(AzureResource.vm, self.config.metrics_vm_resource_group, vmname)
 
                 # write the metrics out to the storage queue
-                vmmetricsSerialized = pickle.dumps(vmmetrics)
-                self.storage_service_queue.put_message(self.config.metrics_queue_name, vmmetricsSerialized)
+                self.write_metrics_to_storage(vmmetrics)
 
             except Exception as ex:
-                self._log_exception(ex, self.capture_vm_metrics.__name__)
\ No newline at end of file
+                self._log_exception(ex, self.capture_vm_metrics.__name__)
+
+    def capture_vmss_metrics(self):
```

Comment:
Add this call to scheduler.py

Reply:
Ok, I am going to complete this PR and do that in another commit.